main.go

```diff
@@ -137,6 +139,9 @@ func main() {
 		AlertSMTPFrom:                envVars[alertSMTPFromAddrEnvVarName],
 		DeploymentType:               envVars[deploymentTypeEnvVarName],
 		CustomerNotificationHTMLPath: "templates/customernotification.html",
+		RhobsSecretName:              fmt.Sprintf("%v-prom-remote-write", addonName),
+		RhobsApiUrl:                  envVars[rhobsApirUrl],
+		RhSSOTokenUrl:                envVars[rhssoTokenUrl],
```

Comment:
We do not read env var directly here. Please read them as part `readEnvVars` function

Reply:
Not reading them directly, here i am setting the vars into the reconciler, changed naming of vars